src/components/players/Players.vue

```diff
@@ -42,28 +44,50 @@
             @blur="date = parseDate(dateFormatted)"
           ></v-text-field>
 
-          <v-date-picker  v-model="date" no-title @input="menu1 = false"></v-date-picker>
+          <v-date-picker v-model="date" no-title @input="menu1 = false"></v-date-picker>
+          
         </v-menu>
+        <p class="select-date" @click="openDatePicker">
+          {{ selectDateTitle }}
+          <v-icon color="white">chevron_right</v-icon>
+        </p>
       </div>
 
     </v-layout>
   </v-container>
 </template>
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
-<style scoped>
+<style lang="scss">
+.menu__content .menuable__content__active {
+  min-width: 200px;
+  top: 673px;
+  left: 192px;
+  transform-origin: left top 0px;
+  z-index: 8;
+}
+
 .new-player {
   align-self: flex-end;
 }
 
 .date-picker {
   max-width: 200px;
   width: 100%;
+  position: relative;
+}
+
+.select-date {
+  color: white;
+  cursor: pointer;
+  position: absolute;
```

Comment:
Hi man, thank's for review. I need this for customizing the original datepicker because he has a very different design. And I need this position to position the popup from original datepicker. I test here in chrome and this not happen.

Reply:
Can you test on Firefox too, please? It behaves completely different from Chrome, I'm not sure why :confused: and we need to support at least these two browsers.

Reply:
I remove the `.fill-height` class from `v-container` and problem solved.

Reply:
@arthurdenner Can you approve?

Reply:
Sorry for the wait, @valterbarros.

Thanks for fixing this. :tada:

Reply:
No man never mind, thanks :tada:

Reply:
And I want work more with you because you are a good coder. I have a doubt how you commit in my branch? In some times I need do this in other projects but I don't know how.